substrate/frame/staking/src/slashing.rs

```diff
@@ -320,19 +312,19 @@ fn kick_out_if_recent<T: Config>(params: SlashParams<T>) {
 	);
 
 	if spans.era_span(params.slash_era).map(|s| s.index) == Some(spans.span_index()) {
+		// Check https://github.com/paritytech/polkadot-sdk/issues/2650 for details
 		spans.end_span(params.now);
-		<Pallet<T>>::chill_stash(params.stash);
 	}
 
-	let disable_without_slash = params.disable_strategy == DisableStrategy::Always;
-	add_offending_validator::<T>(params.stash, disable_without_slash);
+	add_offending_validator::<T>(&params);
 }
 
 /// Add the given validator to the offenders list and optionally disable it.
```

Comment:
This rustdoc needs updating perhaps? There's no offenders list anymore, no more `OffendingValidatorsThreshold ` and no more fore era afaik.

Reply:
Done